garble.py

```diff
@@ -70,7 +73,22 @@ def validate_clks(clk_files, metadata_file):
 
 def garble_pii(args):
     secret_file = Path(args.secretfile)
-    source_file = Path(args.sourcefile)
+
+    if args.sourcefile:
+        source_file = Path(args.sourcefile)
+    else:
+        oldest_ts = datetime.fromtimestamp(0)
```

Comment:
I think this is supposed to be `newest`? The code looks right, just the variable name is backwards. Same thing in households.py

Reply:
yes and I'm happy to change the name. I chose "oldest" because we're looking for the 'largest' timestamp/furthest from 0 but understand the confusion.